app/models/proposal.rb

```diff
@@ -80,7 +97,17 @@ def users
   # returns the Approval
   def add_approver(email)
     user = User.for_email(email)
-    self.approvals.create!(user_id: user.id)
+    approval = self.approvals.create!(user_id: user.id)
+    approval
+  end
+
+  def initialize_approvals()
+    if self.linear? && self.approvals
```

Comment:
I thought they'd evaluate the same, but yep, I meant `self.approvals.any?` Will update.

This check is necessary as we only want the _first_ linear approval to be marked as `actionable` -- the others will enter this state as approvals come in.